README.md

```diff
@@ -6,10 +6,19 @@ Starter project intended to be a quick start for writing NPL.
 
 Clone the project using:
 
```

Comment:
you should remove this line now :)

Reply:
thank you for the feedback. I've removed the line. Please have a look on it. Thank you